### PR TITLE
Fixed bugs

### PR DESCRIPTION
**Pull request by cleorenaud**
**Corresponding issue : #153** 

With this PR the `UserProfileFollowers`, `UserProfileFollowing` and `UserProfileFriend` view should not have the bugs from the issue anymore

### CODE REVIEW

File: app/src/main/java/com/example/triptracker/view/Navigation.kt [hunk 1]
```diff
@@ -22,8 +22,6 @@ object Route {
   const val FOLLOWING = "following"
   const val MYTRIPS = "mytrips"
   const val SETTINGS = "settings"
-  const val FOLLOWERS = "followers"
-  const val FOLLOWING = "following"
```
Comment (JCHAVEROT) [inline app/src/main/java/com/example/triptracker/view/Navigation.kt:26]: If I'm not mistaken you can add them back now

File: app/src/main/java/com/example/triptracker/view/profile/FriendListView.kt [hunk 2]
```diff
@@ -65,175 +65,200 @@ fun FriendListView(
...
+                                verticalArrangement = Arrangement.Center
+                            ) {
                                 Text(
-                                    text = "${friend.name} ${friend.surname}",
+                                    text = friend.username,
                                     style =
+                                    TextStyle(
+                                        fontSize = 16.sp,
+                                        lineHeight = 16.sp,
+                                        fontFamily = FontFamily(Font(R.font.montserrat)),
```
Comment (JCHAVEROT) [inline app/src/main/java/com/example/triptracker/view/profile/FriendListView.kt:141]: I believe that 'fontFamily = Montserrat' works also

File: app/src/main/java/com/example/triptracker/view/profile/FriendListView.kt [hunk 3]
```diff
@@ -65,175 +65,200 @@ fun FriendListView(
...
-                  if (!isToggled) "Follow" else "Following"
-                },
+            // Display the appropriate button text based on whether we are prompting
+            // followers, following or profiles and whether the button have been toggled or not
+            //
+            if (relationship == Relationship.FOLLOWING || relationship == Relationship.FRIENDS) {
+                if (areConnected) "Following" else "Follow"
+            } else {
+                if (areConnected) "Remove" else "Undo"
+            },
```
Comment (JCHAVEROT) [inline app/src/main/java/com/example/triptracker/view/profile/FriendListView.kt:252]: Good job, it much more understandable now

**Review by JCHAVEROT — APPROVED** [review]
LGTM, thanks for the bugs fix !
Just watch out for the CI not passing, maybe because of the two lines in `Navigation.kt`.

File: app/src/main/java/com/example/triptracker/view/profile/FriendListView.kt (hunk 2 above)
Comment (cleorenaud) [reply app/src/main/java/com/example/triptracker/view/profile/FriendListView.kt]: I didn't know, thanks a lot !

**Pull request merged**